README.md

```diff
@@ -1,7 +1,7 @@
-XtreemFS OpenDMK 1.0-b02-SNAPSHOT Build
+XtreemFS OpenDMK 1.0-b03-SNAPSHOT Build
 =======================================
 
-This project aims at providing `jdmkrt`, `jdmktk` and `jmxremote_optional` in a Maven structure. It is built from the 1.0-b02 Sources of [OpenDMK](https://opendmk.java.net/) using Maven 3.1.1 and Oracle JDK 1.5.0_13-b05, because the [binary plug](https://opendmk.java.net/download/index.html#BinaryComponents) (which this project uses) is built using this version as well.
+This project aims at providing `jdmkrt`, `jdmktk` and `jmxremote_optional` in a Maven structure. It is built from the 1.0-b03 Sources of [OpenDMK](https://opendmk.java.net/) using Maven 3.6.3 and Open JDK 14.0.1 with 1.8 bytecode.
```

Comment:
I believe it's still the `1.0-b02` sources of OpenDMK, with modifications to allow for Java 8. You can remove the version information for Maven, as the binaries deployed at the below mentioned GitHub pages use an outdated version. Looking at the [Travis build log](https://travis-ci.org/github/xtreemfs/opendmk/builds/692096429), it appears to be JDK 1.8. I haven't built the project on my machine using JDK 14, I'm sure you've tested it. If I were to make a suggestion, consider using Java 11 compatibility, as this is the current LTS (and rename the PR and comments accordingly). I'm not too familiar with the changes introduced in Java versions above 11, and since you're aiming for bytecode version 1.8 anyway, I don't see a reason to pin it to a non-LTS version of Java.

Reply:
I made some more changes to the README.md file, but I'm unsure if it's what you are looking for.